### ISSUE TITLE
Decoys in spectral library

### ISSUE THREAD

**Issue by wenbostar** [vdemichev/DiaNN#856]
How does DIA-NN recognize decoys in a spectral library? Is it based on the column "Decoy" as shown below in the library?

|  Decoy | 
|---|
|  No  |
|  No |
|  Yes |

Bo

**Comment by vdemichev** [comment]
Yes, but it's strongly recommended NOT to have any decoys in the spectral library.
